mewarpx/examples/capacitive_discharge.py

```diff
@@ -13,6 +13,7 @@
 
 from pywarpx import picmi
 import pywarpx
+from pywarpx import callbacks
```

Comment:
Why was this added?

Reply:
It'll go away when nruof/thermionic_emission branch is merged